lib/bcdice/game_system/Cthulhu7th.rb

```diff
@@ -17,13 +17,19 @@ class Cthulhu7th < Base
 
       # ダイスボットの使い方
       HELP_MESSAGE = <<~INFO_MESSAGE_TEXT
-        ※コマンドは入力内容の前方一致で検出しています。
         ・判定　CC(x)<=（目標値）
         　x：ボーナス・ペナルティダイス (2～－2)。省略可。
```

Comment:
別件ですが、ボーナス・ペナルティダイスの範囲は撤廃されてます
```suggestion
        　x：ボーナス・ペナルティダイス。省略可。
```

Reply:
ありがとうございます！